components/script/dom/webidls/Bluetooth.webidl

```diff
@@ -23,9 +23,8 @@ interface Bluetooth {
 //  [SecureContext]
 //  readonly attribute BluetoothDevice? referringDevice;
 //  [SecureContext]
-//  Promise<BluetoothDevice> requestDevice(RequestDeviceOptions options);
   [Throws]
```

Comment:
This doesn't need to be here anymore, or most (all?) of the other ones in other bluetooth webidl files.